include/tester/vtcp/tcp/TCPListener.hpp

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include<iostream>
+#include<functional>
+
+#include<boost/asio.hpp>
+#include<boost/bind/bind.hpp>
+
+using boost::asio::ip::tcp;
+
+class TCPListener
+{
+public:
+
+    TCPListener(int port, boost::asio::io_context & io_context,
```

Comment:
change type of port to `std::uint16_t` and consider using a const modifier as a prefix (if possible)

Reply:
done